### PR TITLE
LITE-18627 Added prefetch count for RabbitMQ

### PR DESCRIPTION
**Pull request by net-free**

### CODE REVIEW

File: dj_cqrs/utils.py [hunk 1]
```diff
@@ -33,3 +33,37 @@ def get_expires_datetime():
...
+    if max_size is not None and max_size <= 0:
+        logger.warning(
+            "Settings delay_queue_max_size=%s is invalid, using default %s.",
+            max_size, DEFAULT_DELAY_QUEUE_MAX_SIZE,
+        )
+        max_size = DEFAULT_DELAY_QUEUE_MAX_SIZE
+    return max_size
+
+
+def get_rabbitmq_prefetch_count():
```
Comment (maxipavlovic) [inline dj_cqrs/utils.py:58]: Hm, I don't think that it makes sense to put something transport specific (rabbit_mq prefetch count) into common utils.
I would suggest putting it somewhere closer to transport implementation.

**Review by maxipavlovic — CHANGES_REQUESTED** [review]

File: dj_cqrs/utils.py (hunk 1 above)
Comment (net-free) [reply dj_cqrs/utils.py:58]: In general all utils now are rabbitmq specific.
I don't like the idea of keeping every function as a static/class method.
My suggestion - rename it to get_prefetch_count.

Comment (maxipavlovic) [reply dj_cqrs/utils.py:58]: Ok

**Review by maxipavlovic — APPROVED** [review]

**Review by d3rky — APPROVED** [review]

**Comment by d3rky** [comment]
Tests are passed - some issues with uploading coverage

**Comment by Szejdi** [comment]
Hey,

I've found that this Pull Request is breaking `cqrs_dead_letters retry` command.

`_get_consumer_settings` used to return 2 arguments and with this change it start to return tuple of 3,
which is breaking `init_broker` method in above mentioned command.
Exact line:
```
queue_name, dead_letter_queue_name = RabbitMQTransportService.get_consumer_settings()
```
is raising `ValueError: too many values to unpack (expected 2)`

**Comment by maxipavlovic** [comment]
@Szejdi thank you very much. We will fix that!

**Comment by maxipavlovic** [comment]
@Szejdi fixed in [1.13.2](https://github.com/cloudblue/django-cqrs/releases/tag/1.13.2)

**Pull request merged**